setup/native-script.rb

```diff
@@ -114,11 +114,30 @@ def install(program_name, message, script, run_as_root = false, show_all_option
 puts "Configuring your system for Android development... This might take some time, please, be patient."
 # Note that multiple license acceptances may be required, hence the multiple commands
 # the android tool will introduce a --accept-license option in subsequent releases
+error_msg = "There seem to be some problems with the Android configuration"
+
 android_executable = File.join(ENV["ANDROID_HOME"], "tools", "android")
-execute("echo y | #{android_executable} update sdk --filter platform-tools --all --no-ui", "There seem to be some problems with the Android configuration")
-execute("echo y | #{android_executable} update sdk --filter tools --all --no-ui", "There seem to be some problems with the Android configuration")
-execute("echo y | #{android_executable} update sdk --filter android-23 --all --no-ui", "There seem to be some problems with the Android configuration")
-execute("echo y | #{android_executable} update sdk --filter build-tools-23.0.2 --all --no-ui", "There seem to be some problems with the Android configuration")
-execute("echo y | #{android_executable} update sdk --filter extra-android-m2repository --all --no-ui", "There seem to be some problems with the Android configuration")
+execute("echo y | #{android_executable} update sdk --filter platform-tools --all --no-ui", "#{error_msg}")
+execute("echo y | #{android_executable} update sdk --filter tools --all --no-ui", "#{error_msg}")
+execute("echo y | #{android_executable} update sdk --filter android-23 --all --no-ui", "#{error_msg}")
+execute("echo y | #{android_executable} update sdk --filter build-tools-23.0.2 --all --no-ui", "#{error_msg}")
+execute("echo y | #{android_executable} update sdk --filter extra-android-m2repository --all --no-ui", "#{error_msg}")
+
+puts "Do you want to install Android emulator? (y/n)"
```

Comment:
Same remark as in the powershell script - it's better to use the `install` function so that the user may install everything without a great number of questions